doc/Type/Code.pod6

```diff
@@ -145,6 +145,12 @@ Returns a list of candidates that can be called with the given
 L<Capture|/type/Capture>.  Since C<Code> objects do not have any multiple
 dispatch, this either returns a list with the object, or an empty list.
 
+    my $single = \'a';         # a single argument Capture
+    my $plural = \'a', 42;     # a two argument Capture
```

Comment:
Maybe add a () around the constants? This yields:
```
Useless use of constant integer 42 in sink context (lines 1, 1)
```

Reply:
D'oh! I changed that when been testing my code, but not in actual source. Thanks, will fix now...